lib/request-options.js

```diff
@@ -36,7 +36,7 @@ module.exports = (libOptions, logger) => {
         // Set the timeout on the request to a minute. The thought
         // behind this is just setting it to something sort of long,
         // in case Shelf is under a large load.
-        options.timeout = 1000;
+        options.timeout = 60000;
```

Comment:
Can this be defaulted to a minute but set with libOptions? Just incase someone wanted to change it?